stuff2sort/System.py

```diff
@@ -0,0 +1,1150 @@
+#!/usr/bin/env python
+
+#   // INITIAL DESCRIPTION //
+# XX AJ I don't know what to put here because the module only has this one class. What is the difference between the module description and the class description here?
+"""Short Module Description; Reference To Readme"""
+
+#   // MEATDATA //
+__author__ = 'Alina Jansen'
+__date__ = '2024-03-25'
+
+#   // IMPORTS //
+
+#   Imports Of Existing Libraries
+import re
+import os 
+import math
+import numpy as np
+
+#   Imports From Existing Libraries
+
+#   Imports Of Custom Libraries
+import Generators.GeneratorGeometries as geometry
+
+#   Imports From Custom Libraries
+from Logging.Logger import Logger
+from Generators import generate_pcf_from_top as make_pcf
+from Generators._helper import _flatten, get_linkatoms_ang
+
+#   // TODOS & NOTES //
+#   TODO:
+#   - Add the option to read atom input from a list (if that's what we want, wait for Florians opinion)
+#   - Add logger
+#   NOTE:
+
+#   // CLASS & METHOD DEFINITIONS //
+class SystemInfo():
+
+    '''
+    This Class Reads And Stores Information About The System
+    '''
+
+    def __init__(self, input_dict) -> None:
```

Comment:
General and important note: we want to name variables the following way: <datatype>_<specification>_<detailed_specification>. As an example: 

instead of 'self.input_dict', it should be 'self.dict_input'; additionally, you can further specify which input, as we deal with a lot of different 'input types'.

Reply:
Understood (: